### DISCUSSION TITLE
Can not access to dashboard after installation

### DISCUSSION THREAD

**Discussion by marctaule** [Laravel-Backpack/community-forum#966] [Q&A (Help)] [answered]
I just bought the backpack pro and the devtools. I've followed all the instructions from the videos and text, and when I finish the installation and access /admin, it redirects me to /admin/login but I get a 500 error. I've been searching but can't find what it could be. 

![Screenshot 2024-04-25 at 19 56 17](https://github.com/Laravel-Backpack/community-forum/assets/46464634/4a051c27-4b0b-4d21-b828-d94bba757678)

Can anyone give me some guidance? 😭

**Comment by jcastroa87** [comment]
Hello @marctaule 

The installation finish without any problems? include basset configuration?

Can you share with me the output of:
```
php artisan backpack:version
```

Did you check the error log on storage?

I will be waiting for more details to help you.

Cheers.

**Reply by marctaule** [reply, marked as answer]
Thanks @jcastroa87 for your answer. I don't have any problem in my installation. Almost everything goes well, basset included...
I got this error. It says DONE but also ERROR.

Install the Tabler theme ............................................................................................... DONE
.. ERROR
  │ For further information please check the log file.
  │ You can also follow the manual installation process documented on Github.

In laravel.log I see: local.ERROR: sh: line 0: exec: composer: not found

This help me! I've installed the theme manually and now it works :)

Thanks again!
